plugins/mcreator-localization/help/default/entity/data_type.md

```diff
@@ -0,0 +1 @@
+This is the type of the entity's variable. It can be a number, logic or String value.
```

Comment:
Idk why 'string' word is not lowercased here outside Java code, also wondering why you are using 'type' and 'value' in the same context.
```suggestion
This option controls the type of this entity variable. Supported types are logic, number and string variables.
```